integration-tests/tests/soroban.client.ts

```diff
@@ -0,0 +1,54 @@
+import { Server, Contract, TimeoutInfinite, TransactionBuilder, Keypair, xdr, SorobanRpc } from "soroban-client";
+import "./soroban.config";
+
+export class SorobanClient {
+    client: Server;
+
+    constructor() {
+        this.client = new Server(process.env.SOROBAN_RPC_URL, {
+            allowHttp: true
+        });
+        this.client.getHealth();
+    }
+
+    async sendTransaction(
+        contractId: string,
+        method: string,
+        signer: string,
+        secret: string,
```

Comment:
It is more convenient to wrap signer and secret into keypair struct or use some exists